.github/workflows/create-demo-clusters.yml

```diff
@@ -25,6 +25,12 @@ on:
       workflow-ref:
         description: Ref of the called workflow
         type: string
+      central-name:
+        description: The name of the long running cluster where the central deployment is run
+        type: string
+      secured-cluster-name:
+        description: The name of the long running cluster where collector is run
+        type: string
```

Comment:
Make sure that we can easily see which secure cluster is running true collector and which is running sensor with fake workload.

Reply:
These have been changed to cluster-with-fake-load-name and cluster-with-real-load-name